cl/favorites/api_serializers.py

```diff
@@ -46,7 +46,11 @@ class Meta:
 
 
 class PrayerSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
-    user = serializers.HiddenField(default=serializers.CurrentUserDefault())
+    # Use HiddenField for API creation, but still include in serialized output
+    # by setting it in fields. This allows webhooks to include the user ID.
+    user = serializers.PrimaryKeyRelatedField(
+        read_only=True, default=serializers.CurrentUserDefault()
+    )
```

Comment:
I think we should keep using `serializers.HiddenField` and exclude the `user` key from the webhook. This ID will always correspond to the user receiving the webhook, so including it doesn’t add any value, they’d just be getting their own ID. We don’t track who purchased the document or granted the prayers, so there’s no additional reason to include it.